### PR TITLE
Add Sadbaby (BEP20)

### PR DESCRIPTION
**Pull request by sadbabytoken**
Adding info for BEP20 token 'Sadbaby'.

PR created by <a href="https://assets.trustwallet.com">TW Assets Management</a>

### CODE REVIEW

**Comment by sadbabytoken** [comment]
My payment no 1679 has been made

**Comment by cryptomanz** [comment]
Hello,
- As per our policy, brand new tokens are not included, there is a minimum requirement of 5000 holder addresses and 10000 transactions, in order to filter out scam tokens.
See the criteria at (mentioned in the first message by the bot): https://community.trustwallet.com/t/how-to-submit-a-token-logo/3863).
Closing PR for now. Please come back to us when all criteria are met.

**Comment by Cryptocool1** [comment]
hello,
• What is the purpose of this token? How will it benefit Trust Wallet users?
 • How is the token being distributed? Are you conducting an airdrop?
 • Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?
 • Is the token listed on a centralized/decentralized exchange?
 • Is a Smart Contract audit completed? Where can we find this information?

**Comment by sadbabytoken** [comment]
hello sadbaby token is a cryptocurrency created for babies with sma.  The aim is to heal babies with Sma by donating to them.  sadbaby token is traded on azbit exchange, indoex exchange and latoken exchange.  5000 thousand holders have been completed.  Yesterday we attended a big organization in global advertisements and live broadcast.
https://www.instagram.com/tv/CRUpg7_g68T/?utm_medium=copy_link

**Comment by cryptomanz** [comment]
hello,
can I have proofs of donating to babies?

**Comment by cryptomanz** [comment]
• Is a Smart Contract audit completed? Where can we find this information?

**Comment by sadbabytoken** [comment]
https://t.me/sadbabyglobal

**Comment by sadbabytoken** [comment]
Hello, you can see our donations and works by entering our Telegram channel 🙏 We are waiting for the happy news for our logo from you.  we thank you

**Comment by sadbabytoken** [comment]
https://bscscan.com/token/0x4E84e47bBABD89835e2e9569d69A6d465DE38122

**Comment by sadbabytoken** [comment]
https://poocoin.app/tokens/0x4e84e47bbabd89835e2e9569d69a6d465de38122


 https://t.me/IndoEx/1164https://twitter.com/Indoex_LTD/status/1412760074284531714?s=…https://t.me/azbit_news/630

https://latoken.com/exchange/SDBY_TRX

**Comment by Cryptocool1** [comment]
• Is a Smart Contract audit completed? Where can we find this informatio

**Comment by sadbabytoken** [comment]
• Is a Smart Contract audit completed? Where can we find this information

i don't understand your question i'm sorry

**Comment by sadbabytoken** [comment]
yes, you can contact Latoken exchange officials.

**Comment by Cryptocool1** [comment]
> yes, you can contact Latoken exchange officials.

Can you provide us audit?

**Comment by sadbabytoken** [comment]
Https://t.me/@AzizF_LA

**Comment by sadbabytoken** [comment]
latoken Exchange official Kuc verification was carried out on the Latoken exchange.  and everything is under control.  I frankly do not fully understand your question.  i do my best to help you

**Comment by sadbabytoken** [comment]
latoken Exchange official Kyc verification was carried out on the Latoken exchange.  and everything is under control.  I frankly do not fully understand your question.  i do my best to help you

**Comment by Cryptocool1** [comment]
I’m asking about smart contract audit, not about telegram

**Comment by sadbabytoken** [comment]
not yet audited and will be audited in the future

**Comment by sadbabytoken** [comment]
Could you please confirm our logo.  Everyone is supporting and trying with all their might for our babies with SMA.  We are waiting for news for coinmarketcap and waiting for you to upload our logo.  you are close to a light for our babies.  I made your payment and sent the information you requested.  I sent donation receipts.  If there is anything you want, I can send you.  Thank you very much for taking care of us.

**Comment by cryptomanz** [comment]
Hello,
- Awaiting for smart contract audit.

**Comment by sadbabytoken** [comment]
Is there something I should do.  what can i do for you.  You said you are waiting for smart contract control.  will you check

**Comment by Cryptocool1** [comment]
> Is there something I should do. what can i do for you. You said you are waiting for smart contract control. will you check

yes, we are waiting for audit

**Comment by sadbabytoken** [comment]
Will you perform the inspection?

**Comment by Cryptocool1** [comment]
as I mention we are waiting for your smart contract audit

**Comment by sadbabytoken** [comment]
Will you do this inspection?

**Comment by sadbabytoken** [comment]
do you want something from me

**Comment by Cryptocool1** [comment]
Don’t really understand you. 
As we mentioned before, we are awaiting the smart contract audit from you

**Comment by sadbabytoken** [comment]
Can we learn the content of the contract, can you give us an example?  Our lawyer wants information on the content

**Comment by cryptomanz** [comment]
> Can we learn the content of the contract, can you give us an example? Our lawyer wants information on the content

We just want to see your smart contract audit.
Reply to this message when it will be ready.

**Comment by cryptomanz** [comment]
Hello,
please create new PR when smart contract audit will be ready.
make sure to mention this PR, so that we can see that fee has been paid already here.

**Comment by sadbabytoken** [comment]
hello i want to share with you the link of your smart contracts
https://bscscan.com/address/0x4e84e47bbabd89835e2e9569d69a6d465de38122#code

**Comment by sadbabytoken** [comment]
#11950

**Comment by sadbabytoken** [comment]
#12038  
https://1e62d418-03b1-4053-96ed-24b2d39e6f74.filesusr.com/ugd/5ed437_79a0cb4f915841a7b61a88e036fb132d.pdf

**Comment by sadbabytoken** [comment]
https://www.tech-audit.org/

**Comment by sadbabytoken** [comment]
https://1e62d418-03b1-4053-96ed-24b2d39e6f74.filesusr.com/ugd/5ed437_79a0cb4f915841a7b61a88e036fb132d.pdf

File: info.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+" name " : " Sadbaby Token " , 
+" web sitesi " : " https://sadbaby.org/ , 
+" description " : " SadBaby Token, öncelikle SMA hastası bebeklerimizin arabası hala sevdikleri bir yer olarak yaşamalarını sağlamak için bir bağış bağışladı " jetonudur, 
+" kaşif " : " https://bscscan.com/token/0x4e84e47bbabd89835e2e9569d69a6d465de38122 " ,
```
Comment (sadbabytoken) [inline info.json]: 0x3ad405ef7aea80ccb41beef0a74510e18feef190

Comment (sadbabytoken) [reply info.json]: it has been abandoned

**Comment by sadbabytoken** [comment]
OLD_CONTRACT_ADDRESS 0x4e84e47bbabd89835e2e9569d69a6d465de38122

NEW_CONTRACT_ADDRESS 0x3ad405ef7aea80ccb41beef0a74510e18feef190

File: info.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+" name " : " Sadbaby Token " , 
+" web sitesi " : " https://sadbaby.org/ , 
+" description " : " SadBaby Token, öncelikle SMA hastası bebeklerimizin arabası hala sevdikleri bir yer olarak yaşamalarını sağlamak için bir bağış bağışladı " jetonudur, 
+" kaşif " : " https://bscscan.com/token/0x4e84e47bbabd89835e2e9569d69a6d465de38122 " , 
+" type " : " BEP20 " , 
+" , 
+" jetonudur " : " SDBY " , " ondalık say " : 9 , 
+" durum " : " aktif " , 
+" id " : " 0x4e84e47bbabd89835e2e9569d69a6d465de38122 "
```
Comment (sadbabytoken) [inline info.json]: 0x3ad405ef7aea80ccb41beef0a74510e18feef190

**Review by sadbabytoken — COMMENTED** [review]
0x3ad405ef7aea80ccb41beef0a74510e18feef190

**Pull request merged**